apps/fares/lib/fares.ex

```diff
@@ -22,6 +22,17 @@ defmodule Fares do
   @foxboro_reverse_commute ~w(741 743 745 750 752 754 756)
   @foxboro_reverse_commute_set MapSet.new(@foxboro_reverse_commute)
 
+  @free_sl1_fare %Fares.Fare{
```

Comment:
Why the need for this new type? I thought trips from the airport already correctly had a free fare associated with them?